src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -2015,7 +2019,7 @@ public override BoundNode VisitMethodGroup(BoundMethodGroup node)
 
             //if (this.State.Reachable) // PROTOTYPE(NullableReferenceTypes): Consider reachability?
             {
-                this.State.ResultType = null;
+                _result = null;
```

Comment:
>_result = null [](start = 16, length = 14)

Strange to see null assigned to a struct. Does this work because of implicit conversion?

Reply:
Yes, the implicit conversion is used.